client/app/distressButton/distressButton.js

```diff
@@ -12,7 +12,7 @@ angular.module('distress')
     $http({
       method: 'POST',
       url: '/sms/text/',
-      data: username
+      data: {username:username}
```

Comment:
Sending in an object so no change needed to bodyparser and increases readability of code in smsController